md.py

```diff
@@ -281,11 +293,30 @@ def get_num_disks(self):
         return int((self._sysfs / "raid_disks").read_text().strip())
 
     def get_disks(self):
-        for d in range(self.get_num_disks()):
-            disk = (self._sysfs / f"rd{d}" / "block").readlink().name
-            yield f"/dev/{disk}"
+        return self.devs
+
+    def _zero_disk(self, dev):
+        subprocess.check_call(["dd", "if=/dev/zero", f"of={dev}",
+                               "bs=1M", f"count={self._size_to_zero}",
+                               "oflag=direct"],
+                              stderr=subprocess.DEVNULL)
+
+    def zero_all_disks(self, size_to_zero):
+        self._stop_and_create_disks()
+
+        count = size_to_zero + self.MAX_SUPERBLOCK_SZ
+        count = (count + (1 << 20) - 1) >> 20
+        self._size_to_zero = count
```

Comment:
Should we add a dev size limit here in case we have very small ram disks or large zeroing ranges? Or id dd ok with count*bs > dev size?

Reply:
On second thought, we're using a spare disk as the veification scratch pad so end of disk tests will need special consideration anyway, this is good as is.